src/test/resources/org/jenkinsci/plugins/casc/PluginInstallTest.yml

```diff
@@ -0,0 +1,9 @@
+plugins:
+  proxy:
```

Comment:
That questionable UI choice has been debated before and may not be final.

I'd place it outside 'plugins' to prevent confusion: This proxy config is used for a lot more than just plugins.

Reply:
please excuse me if it's a dumb question, but where do we configure it in UI today?

Reply:
Today it's under `Manage Plugins` -> `Advanced` tab... 

@daniel-beck I'll have to ponder a bit on this. Wouldn't not following the UI cause even more confusion for people trying out the plugin?  It would be simple to move proxy to a root element, but we'd be breaking the convention we set up.

Reply:
I'm not 100% on board with this 'mirror the UI' metaphor, but there's a strong argument that the proxy config does not belong to Manage Plugins, as it extends to SCMs, other network features, etc.

See KK's comment to [JENKINS-8885](https://issues.jenkins-ci.org/browse/JENKINS-8885), for example, as well as a bunch of issues related to "X plugin does not apply proxy settings".

> This is actually by design — despite its position in the UI, the proxy setting does apply to all the HTTP access. This is consistent with most applications, where you configure a proxy once and it affects the application as a whole. I think it's confusing if each subsystem in Jenkins start asking for a separate proxy setting.

I think the main reason it's still there is that just nobody has gotten around to moving it to the global configuration, eliminating any confusion about the scope of this setting. I cannot find an issue tracking that at the moment.

Reply:
Of course the clean solution would be to have a core change move the setting to Configure System, and move the proxy config into the appropriate YAML section here 😄

Reply:
@daniel-beck I'd be ok to add the proxy attribute to the `jenkins` element in anticipation that the configuration will be eventually moved to that place. We can document the exception in the Readme.  

@ndeloof and @ewelinawilkosz you ok with that?

Reply:
how is it now - does it "naturally" belongs under the "plugins" from the implementation point of view or did we use some glue code to make it that? 
Does it make sense to move it whenever we agree it's supposed to be? or is it too much mess - having plugins as root element and proxy under jenkins, especially if we don't know where and when?

Reply:
I was lost looking for it, that's why I asked, but I expected it under 'Configure System' which means 'jenkins' in yaml, so I'd be ok with your propoasl @MadsNielsen 
would like to hear @ndeloof opinion though

Reply:
@daniel-beck a key driver for configuration-as-code design is to mimic the web UI. This is sort of a "natural documentation" for end-users. The fact the proxy configuration is misplaced in PluginManager, is a core design issue. If at some point this is moved to some `NetworkGlobalConfiguration` we would for sure adopt it.